.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '5'
+- '6.10.3'
```

Comment:
I'd also like to get us on current stable node versions. 🆙😃 

If each lib that depends on CF works on node 6, then the way to resolve this is by opening PRs to upgrade each lib. However I would suggest doing that as a separate piece of work.